### PR TITLE
Remove dead code

### PR DESCRIPTION
**Pull request by Pessimistress**

### CODE REVIEW

File: src/mercator-viewport.js [hunk 1]
```diff
@@ -187,8 +189,8 @@ export default class MercatorViewport {
    *   Specifies a point on the sphere to project onto the map.
    * @return {Array} [x,y] coordinates.
    */
-  projectFlat([x, y], scale = this.scale) {
-    return this._projectFlat(...arguments);
+  projectFlat(xyz, scale = this.scale) {
+    return xyz;
```
Comment (ibgreen) [inline src/mercator-viewport.js]: Not sure it matters, but what does returning xyz here mean? 

Should this assert? If just a dummy return value, maybe a comment?

File: src/mercator-viewport.js [hunk 2]
```diff
@@ -200,45 +202,7 @@ export default class MercatorViewport {
...
-    return this._unprojectFlat(...arguments);
+    return xyz;
   }
 
-  getMatrices({modelMatrix = null} = {}) {
-    let modelViewProjectionMatrix = this.viewProjectionMatrix;
-    let pixelProjectionMatrix = this.pixelProjectionMatrix;
-    let pixelUnprojectionMatrix = this.pixelUnprojectionMatrix;
-
-    if (modelMatrix) {
```
Comment (ibgreen) [inline src/mercator-viewport.js]: We are dropping modelMatrix support, assuming it was never documented in this module

File: src/mercator-viewport.js (hunk 1 above)
Comment (Pessimistress) [reply src/mercator-viewport.js]: Yes, this is a dummy value. `projectFlat` is overridden by `PerspectiveMercatorViewport`.

File: src/mercator-viewport.js (hunk 2 above)
Comment (Pessimistress) [reply src/mercator-viewport.js]: We never had `modelMatrix` support - This method is copied from deck.gl and not called anywhere.

**Pull request merged**